frontend/tests/unit/mixins.shootStatusCredentialRotation.spec.js

```diff
@@ -4,8 +4,7 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
-import { shootStatusCredentialRotation } from '@/mixins/shootStatusCredentialRotation'
-import { rotationTypes } from '@/utils/credentialsRotation'
+import { shootStatusCredentialRotation, rotationTypes } from '@/mixins/shootStatusCredentialRotation'
```

Comment:
Use rot
```suggestion
import shootStatusCredentialRotation from '@/mixins/shootStatusCredentialRotation'
```
Use rotationTypes as `shootStatusCredentialRotation.rotationTypes` or on the instance as `this.$options.rotationTypes`